scaffolding-chef-inspec/lib/scaffolding.sh

```diff
@@ -141,20 +141,12 @@ EOF
         "file": "{{pkg.svc_path}}/logs/inspec_last_run.json"
       }{{#if cfg.automate.enable ~}},
       "automate" : {
-        "url": "{{cfg.automate.server_url}}/data-collector/v0/",
+        "url": "{{cfg.automate.server_url}}",
         "token": "{{cfg.automate.token}}",
         "node_name": "{{ sys.hostname }}",
         "verify_ssl": false
       }{{/if ~}}
     }
-    {{#if cfg.automate.enable }},
-    "compliance": {
-     "server" : "{{cfg.automate.server_url}}",
-     "token" : "{{cfg.automate.token}}",
-     "user" : "{{cfg.automate.user}}",
-     "insecure" : true,
-     "ent" : "automate"
-    }{{/if }}
```

Comment:
I can not find in InSpec where this configuration is used. https://www.inspec.io/docs/reference/config/. By removing it I was able to run InSpec.

Reply:
You're right in that this is left-overs from a previous pattern (pulling profiles from the Automate server at runtime). I'm leaning towards removing it entirely, but this will require some deeper discussion with other maintainers.